components/root_layout/root_layout_body.vue

```diff
@@ -91,6 +106,10 @@ export default {
         { 'root-layout__body--invert': this.sidebarPosition === ROOT_LAYOUT_SIDEBAR_POSITIONS.RIGHT },
       ];
     },
+
+    mainHeight () {
+      return `calc(100vh - (${this.headerHeight} + ${this.footerHeight}))`;
```

Comment:
The `100vh` is very tricky on mobile devices. Safari or Android browsers handle it very inconsistently (e.g. not reducing the viewport when [adding some browser bars](https://chanind.github.io/javascript/2019/09/28/avoid-100vh-on-mobile-web.html)).

A more reliable calculation instead of `100vh` may be to use `window.innerHeight` as suggested [here](https://dev.to/nirazanbasnet/dont-use-100vh-for-mobile-responsive-3o97).

Reply:
updated to use innerHeight in a resizeObserver